src/RecoveryServices/RecoveryServices.SiteRecovery/ReplicationProtectedItem/SetAzureRmRecoveryServicesAsrReplicationProtectedItem.cs

```diff
@@ -130,6 +130,38 @@ public class SetAzureRmRecoveryServicesAsrReplicationProtectedItem : SiteRecover
         [Parameter]
         public string RecoveryAvailabilitySet { get; set; }
 
+        /// <summary>
+        /// Gets or sets the SQL Server license type to the machine to in the event of a failover.
+        /// </summary>
+        [Parameter]
+        [ValidateNotNullOrEmpty]
+        [ValidateSet(
+            Constants.NoLicenseType,
```

Comment:
>NoLicenseType [](start = 22, length = 13)

Same as above

Reply:
Need to check on this once if we need to consider NotSpecified.

---
In reply to: [601549109](https://github.com/AsrOneSdk/azure-powershell/pull/21#discussion_r601549109) [](ancestors = 601549109)